owlkettle/widgets.nim

```diff
@@ -3401,6 +3401,109 @@ renderable AboutDialog of BaseWidget:
         "Art": @["Max Mustermann"]
       }
 
+type ScalePosition* = enum
+  ScaleLeft,
+  ScaleRight,
+  ScaleTop,
+  ScaleBottom
+
+proc toGtk(pos: ScalePosition): GtkPositionType =
+  result = GtkPositionType(ord(pos))
+
+type Mark* = tuple
+  label: Option[string]
+  value: float64
+  position: ScalePosition
+
+renderable Scale of BaseWidget:
+  ## Wrapper for GTK Scale Widget: https://docs.gtk.org/gtk4/class.Scale.html
+  marks: seq[Mark] = @[]
+  min: float64 = 0
+  max: float64 = 100
+  inverted: bool = false
+  showValue: bool = true
+  stepSize: float64 = 5
+  pageSize: float64 = 10
+  value: float64 = 0
+  orient: Orient = OrientX
+  showFillLevel: bool = true
+  precision: int64 = 1
+  valuePosition: ScalePosition
```

Comment:
Also I would personally put `value`, `min` and `max` at the top, because they are probably the most important fields here. → The order the fields appear in the docs is the same as the order they are declared in. So this will improve documentation readability.

Reply:
I'm honestly wondering if "pageSize" should be available. Should I just provide a "multiplier" over stepSize (since that's the intention to use it as a larger "step" than the normal one) ?

Reply:
I think it is generally preferred to stay closer to the GTK widget, so users can just look at the GTK docs.